cmd/telemetrygen/internal/logs/worker.go

```diff
@@ -26,6 +30,23 @@ type worker struct {
 	index          int             // worker index
 }
 
+var reRandom = regexp.MustCompile(`\${random:(\d+)-(\d+)}`)
```

Comment:
For future-proofing, I would first match the 'this is a function' syntax (in this case, something in between `${` and `}`, but see above), and then parse the `random` function, failing if we don't have a `random` function inside. This will allow us to add more functions over time without potentially breaking people each time.

Reply:
This was my initial idea first, but then I wasn't sure, if it's valuable for the community, so I kept it as simple as possible.

I can modify it like this.

Reply:
If we don't do this, then every time we add a new function would imply a breaking change. This way we are reserving the right to implement new functions in the future if we want to.